pyproject.toml

```diff
@@ -0,0 +1,96 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[tool.hatch.version]
+path = "pylhc/__init__.py"
+
+[tool.hatch.build.targets.sdist]
+exclude = [
+  "/.github",
+  "/doc",
+  "/tests",
+]
+
+[tool.hatch.build.targets.wheel]
+packages = ["pylhc"]
+
+[project]
+name = "pylhc"
+readme = "README.md"
+description = "An accelerator physics script collection for the OMC team at CERN."
+authors = [
+    {name = "OMC Team", email = "pylhc@github.com"},  # see zenodo file / commits for details
+]
+license = "MIT"
+dynamic = ["version"]
+requires-python = ">=3.9"
+
+classifiers = [
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: MIT License",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+    "Typing :: Typed",
+]
+
+dependencies = [
+    "numpy >= 1.24",
+    "scipy >= 1.10",
+    "pandas >= 2.1",
+    "matplotlib >= 3.8",
+    "tfs-pandas >= 3.8",
+    "generic-parser >= 1.1",
+    "parse >= 1.15",
+    "omc3 >= 0.15",
+    "jpype1 >= 1.3",
+    "h5py >= 2.9.0",  # same as tfs-pandas, do we need it here?
+    "tables >= 3.6.0",  # same as tfs-pandas, do we need it here?
```

Comment:
Fairly sure it can be removed. Was probably forgotten with version 0.8.0:

```
## Version 0.8.0
[...]
Removed:
 - `hdf5` reader/writer for `tfs` as this is already in `tfs-pandas`
 ```

Reply:
Ooooh correct! Always happy to remove `pytables` from somewhere!